src/helpers/isSearchResultSame.js

```diff
@@ -0,0 +1,27 @@
+// Currently there is no unique id with search result. To match if 2 results are same
+// we check first that they are on same page and then if their coordinates match,
+// then they are considered to be same result
+function isSearchResultSame(searchResultA, searchResultB) {
+  if (!searchResultA || !searchResultB) {
+    return false;
+  }
+  const isOnSamePage = searchResultA.pageNum === searchResultB.pageNum;
+  if (!isOnSamePage) {
+    return false;
+  }
+  const quadA = searchResultA.quads[0];
+  const quadB = searchResultB.quads[0];
+  if (quadA.x1 === quadB.x1 &&
```

Comment:
Problem with the previous implementation was using `Object.values()` which return values in same order as `for...in` loop and the order of values are not guaranteed

> Because the order of iteration is implementation-dependent, iterating over an array may not visit elements in a consistent order. 

This implementation makes sure that we don't have to trust the order

Reply:
Sure, but I was talking about extracting it as it is. No need to change the implementation.

But since the method is short, I think it is clear what it means to have all those identical coordinates.